README.md

```diff
@@ -0,0 +1,87 @@
+Trigger Codemagic workflow
+==========================
+
+GitHub Action to trigger a workflow on Codemagic.
+
+Quick start
+-----------
+
+Add the following configuration to `.github/workflows/main.yml` to trigger Codemagic build on any push event.
+
+    on: push
+
+    jobs:
+      trigger-codemagic-build:
+        runs-on: ubuntu-latest
+        steps:
+          - name: Trigger Codemagic build
+            uses: codemagic-ci-cd/trigger-codemagic-workflow-action@v1.0.0
+            with:
+              app-id: <MY-APPLICATION-ID>
+              workflow-id: <MY-WORKFLOW-ID>
+              token: ${{ secrets.CODEMAGIC_API_TOKEN }}
+
+Required `with` arguments:
```

Comment:
let's have to examples and 1 table with all arguments
- add [optional] or [required] to each description
- for required arguments provide documentation link in description
- for branch/tag arguments provide all description in the table
- instead of `software` list all arguments and add example for each value